docs/section22/Rule22-4.md

```diff
@@ -0,0 +1,58 @@
+
+# CHW&CW - Rule 22-4  
+
+**Rule ID:** 22-4  
+**Rule Description:** For Baseline chilled water loop that is not purchased chilled water and does not serve any computer room HVAC systems, chilled-water supply temperature shall be reset using the following schedule: 44F at outdoor dry-bulb temperature of 80F and above, 54F at 60F and below, and ramped linearly between 44F and 54F at temperature between 80F and 60F.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.9 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule 22-3.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, or 13: `PLACEHOLDER`
+
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+3. Pass Rule 22-3: `if Rule-22-3 == "PASS"`
+
+## Rule Logic:  
+
+- For each HVAC system in B-RMR: `for hvac_b in B-RMR...heating_ventilation_air_conditioning_systems:`
+
+  - Check if HVAC system serves computer room: `if hvac_b.does_serve_computer_room:`
+
```

Comment:
this check will have to be based on the space type of the zones served by the system. need to amend the RDS to find the zones served by the system and the spaces types of each of the zones. If the space type = computer room then the associated system is ignored.

Reply:
The check from Line 32 to 36 was to identify if the CHW loop serves any computer room HVAC system. My understanding is that if the CHW loop serves any computer room HVAC system, the whole loop does not need to reset based on OSA temperature. If this is correct,  I need to update Rule 22-3 to include the same logic here.

Reply:
@xingcx and @mkarpman might make sense to split this into 4 rules
1. If CHW loop serves systems 7,8,12,13 then it is reset based on OAT
2. if CHW loop serves sys 11, it is reset based on cooling load
3. if reset based on OAT, then the required temp setpoints are correct
4. if reset based on cooling load, then chw supp temp = 54F. 
Would this make sense?

Reply:
As discussed during the RDS call, we will have 2 rules: the reset type and the second rule will check the reset settings. Thats how the rules are configured right now.

Reply:
Updated.